cmake/rocksdb_flags.cmake

```diff
@@ -6,42 +6,52 @@ option(WITH_JEMALLOC "build with JeMalloc" OFF)
 if (WITH_JEMALLOC)
   find_package(JeMalloc REQUIRED)
   add_definitions(-DROCKSDB_JEMALLOC -DJEMALLOC_NO_DEMANGLE)
-  include_directories(${JEMALLOC_INCLUDE_DIR})
+  include_directories(${JeMalloc_INCLUDE_DIRS})
 endif()
 
 option(WITH_SNAPPY "build with SNAPPY" OFF)
 if (WITH_SNAPPY)
-  find_package(snappy REQUIRED)
+  find_package(Snappy REQUIRED)
   add_definitions(-DSNAPPY)
-  include_directories(${SNAPPY_INCLUDE_DIR})
+  include_directories(${Snappy_INCLUDE_DIRS})
 endif()
 
 option(WITH_BZ2 "build with bzip2" OFF)
 if (WITH_BZ2)
-  find_package(bzip2 REQUIRED)
+  find_package(BZip2 REQUIRED)
   add_definitions(-DBZIP2)
-  include_directories(${BZIP2_INCLUDE_DIR})
+  if(BZIP2_INCLUDE_DIRS)
```

Comment:
no need to change?

Reply:
There are both DIRS and DIR. DIR is probably from older version.